### ISSUE TITLE
Accessibility concern with <del>

### ISSUE THREAD

**Issue by YariMorcus** [YariMorcus/product-preview-card-component#3]
See [mdn web docs `<del>` page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/del#accessibility_concerns) for more information.

Priority should be concerned as __HIGH__, because __ALL__ applications should be accessible according to Web Accessibility.
